README.md

```diff
@@ -1,25 +1,49 @@
 # bem-twig-extension
-Twig function that inserts static classes into Pattern Lab but adds them to the attributes object in Drupal
+Twig function that inserts static classes into Pattern Lab but adds them to the Attributes object in Drupal
 
-## Usage
+## Usage (4 arguments)
 
-#### Simple block name:
-`<h1{{ bem('title') }}>`
+#### Simple block name (required argument):
+`<h1 {{ bem('title') }}>`
 
 This creates:
 
 `<h1 class="title">`
 
-#### Block with modifiers (array allowing multiple modifiers):
-`<h1{{ bem('title', ({1: 'small', 2: 'red'})) }}>`
+#### Block with modifiers (optional array allowing multiple modifiers):
+`<h1 {{ bem('title', ['small', 'red']) }}>`
 
 This creates:
 
 `<h1 class="title title--small title--red">`
 
-#### Element with modifiers and blockname:
-`<h1{{ bem('title', ({1: 'small', 2: 'red'}), 'card') }}>`
+#### Element with modifiers and blockname (optional):
+`<h1 {{ bem('title', ['small', 'red'], 'card') }}>`
 
 This creates:
 
 `<h1 class="card__title card__title--small card__title--red">`
+
+#### Element with blockname, but no modifiers (optional):
+
+`<h1 {{ bem('title', '', 'card') }}>`
+
+This creates:
+
+`<h1 class="card__title">`
+
+#### Element with modifiers, blockname and extra classes (optional - in case you need non-BEM classes):
```

Comment:
Sorry... last request. This should say "optional array - in case..." to specify that it is an array. Like we did with modifiers.